pkg/translator/azurelogs/resourcelogs_to_logs_test.go

```diff
@@ -239,12 +239,17 @@ var badTimeLogRecord = func() plog.LogRecord {
 
 func TestAsTimestamp(t *testing.T) {
 	timestamp := "2022-11-11T04:48:27.6767145Z"
-	nanos, err := asTimestamp(timestamp)
+	nanos, err := asTimestamp(timestamp, nil, 0)
+	assert.NoError(t, err)
+	assert.Less(t, pcommon.Timestamp(0), nanos)
+
+	timestamp = "11/20/2024 13:57:18"
+	nanos, err = asTimestamp(timestamp, []string{"01/02/2006 15:04:05"}, 0)
 	assert.NoError(t, err)
 	assert.Less(t, pcommon.Timestamp(0), nanos)
 
 	timestamp = "invalid-time"
-	nanos, err = asTimestamp(timestamp)
+	nanos, err = asTimestamp(timestamp, nil, 0)
 	assert.Error(t, err)
 	assert.Equal(t, pcommon.Timestamp(0), nanos)
 }
```

Comment:
please add a case where the parsing fails?

Reply:
added.